examples/pytorch/eeg-gcnn/README.md

```diff
@@ -0,0 +1,48 @@
+# DGL Implementation of EEG-GCNN Paper
+This DGL example implements the EEG-GCNN model proposed in the paper [EEG-GCNN](https://arxiv.org/abs/2011.12107). The original code is [here](https://github.com/neerajwagh/eeg-gcnn).
+
+## All References
+- Paper can also be found on [PMLR](http://proceedings.mlr.press/v136/wagh20a.html).
```

Comment:
Is this one the same as the ArXiv version? If so, we can probably remove it and just mention it in the citation entry at the end.

Reply:
Hi Mufei,

This is a simplified version. The original code has precomputed patient indices. In this example, I removed the dependency on the indices.

Reply:
I see, thanks. Then perhaps you can mention explicitly that this is a simplified version.